tests/cypress/support/commands.js

```diff
@@ -686,10 +643,10 @@ Cypress.Commands.add('checkOlLayer', (args = null) => {
             if (!l.id) {
                 throw new Error(`Invalid layer object ${l}: don't have an id`)
             }
-            if (l.visible == undefined) {
+            if (l.visible === undefined) {
```

Comment:
Here I would not put a `===` because it won't catch `null`, same below for opacity

Reply:
As discussed, I've switch to a logic summarized by "If there's no `visible` or `opacity` key set to the JSON object, I set some default values"